spec/integration/integration.spec.js

```diff
@@ -3,11 +3,45 @@
 const services = require('../helpers/services');
 
 const LONG_TIMEOUT = 30000;
+const AWS = require('aws-sdk');
+
+// Set the region and endpoint in the config for LocalStack
+AWS.config.update({
+  region: 'us-east-1',
+  endpoint: 'http://localhost:4566'
+});
+AWS.config.credentials = new AWS.Credentials({
+  accessKeyId: 'test',
+  secretAccessKey: 'test',
+});
+
+const ssm = new AWS.SSM();
+
+const params = {
+  Name: '/dev/lambda/common/LAMBDA_STAGE',
+  Type: 'String',
+  Value: 'my-value',
+  Overwrite: true
+};
 
 describe('LocalstackPlugin', () => {
 
   beforeEach( () => {
```

Comment:
nit: I think we could also await the `.promise()` on the SSM API call here: (same functionality, but would be slightly shorter)
```
  beforeEach( async () => {
    await ssm.putParameter(params).promise();
    this.service = services.createService({});
  });
```

Reply:
thanks for the hint 🙏  i was actually struggling to set this up correctly.